hypertrace-core-graphql-platform/build.gradle.kts

```diff
@@ -9,7 +9,7 @@ dependencies {
     api("org.hypertrace.core.grpcutils:grpc-client-utils:0.6.1")
     api("org.hypertrace.core.grpcutils:grpc-client-rx-utils:0.6.1")
     api("org.hypertrace.gateway.service:gateway-service-api:0.1.59")
-    api("org.hypertrace.core.attribute.service:caching-attribute-service-client:0.13.3")
+    api("org.hypertrace.core.attribute.service:caching-attribute-service-client:0.13.6")
```

Comment:
The update wasn't really needed since it was a server issue, in the future keeping the revert (or in this case, revert of a revert) clean is clearer to the reviewer.